.github/workflows/test.yml

```diff
@@ -3,7 +3,7 @@ name: Test
 on:
   workflow_dispatch:
   push:
-    branches: [ main ]
+    branches: '**'
```

Comment:
This means "all branches"... which is the same as not restricting branches at all. E.g. you could just remove this line, and it'd always run on any push.